### DISCUSSION TITLE
Unable to get sync footage into a share on my Nas

### DISCUSSION THREAD

**Discussion by phardy2410** [JurajNyiri/HomeAssistant-Tapo-Control#721] [Discuss]
Added network storage to HA
![image](https://github.com/user-attachments/assets/3e2a7aaa-051e-4a4a-92fd-bb4075ba2973)

Set the path in the integration as follows:
![image](https://github.com/user-attachments/assets/d1f40d9a-5cb6-4d1b-ba68-d1ee21052e55)

Fully rebooted HA, started sync and it begins downloading:
![image](https://github.com/user-attachments/assets/640d7d46-7497-45c7-ad6a-80a09f96a7c0)

no files present on the share
![image](https://github.com/user-attachments/assets/91ab28ae-17fd-4bdb-9d91-8ab079072576)

I am sure its something i have / havent done but what? All help appreciated.

**Comment by JurajNyiri** [comment]
The path shown is not correct. It needs to start with `/` and be the path where the remote is mounted inside HA filesystem.

**Comment by JurajNyiri** [comment]
Get https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/5.8.3, that will now check is the path exists before accepting it.

**Comment by phardy2410** [comment]
so given the above screenshots can you guess at the path?

i have tried:

/share/discovault/vault/
/share/discovault.lan/vault/
/share/vault/

after some clues from this thread: https://community.home-assistant.io/t/where-is-network-storage-back-up-mount-path/596925/8

**Reply by phardy2410** [reply]
i asked on that communtiy thread also

**Reply by JurajNyiri** [reply]
No I cannot. 
It depends on your environment. Just ssh in and look inside the folders to find it.

**Reply by phardy2410** [reply]
thats ok, appreicate this isnt an integration issue, its a user one (me)

thank you for your help

**Reply by phardy2410** [reply]
cracked it i think

had the share setup as "media" not "share" so corrected that, found the share in SSH (reddit help) and set the path in the integration as /share/Vault/Cameras/72hours/ and it saved so the integration has permissions (i trust?)

restarted HA

started the sync and its downloading etc etc 

files are now showing
